### PR TITLE
changelog: add 0.2.0 details

### PR DESCRIPTION
**Pull request by gumb0**
I didn't include some very minor PRs like variable renaming etc.

Also didn't include #389 and #390 which will be needed only for type validation.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,56 @@ and this project adheres to [Semantic Versioning].
 ### Added
 
 - API for resolving imported function names into function types. [#318](https://github.com/wasmx/fizzy/pull/318)
+- Validation of stack height at the end of blocks.  [#319](https://github.com/wasmx/fizzy/pull/319) 
+  [#356](https://github.com/wasmx/fizzy/pull/356)
+- Validation of stack height at the end of functions. [#343](https://github.com/wasmx/fizzy/pull/343)
```
Comment (gumb0) [inline CHANGELOG.md:15]: We could group all of stack height validations together

Comment (axic) [reply CHANGELOG.md:15]: Sounds good, just mention these 4 cases in the description.

**Comment by axic** [comment]
Can you also put a summary of tests passing (copy the format from the 0.1.0 changelog)?

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,9 +7,64 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.2.0] — unreleased
```
Comment (axic) [inline CHANGELOG.md:9]:
```suggestion
Firstly, this release implements a lot of validation steps prescribed by the specification, with the exception of type checking.

Secondly, two major optimisations are included: branch resolution is done once in the parser and not during execution; an optimised stack implementation is introduced.
```

or

```suggestion
This release focused on implementing a lot of validation steps prescribed by the specification, and introduces two major optimisations.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,9 +7,70 @@ and this project adheres to [Semantic Versioning].
...
+- Validation of element section. [#350](https://github.com/wasmx/fizzy/pull/350)
+- Validation of export section. [#357](https://github.com/wasmx/fizzy/pull/357)
+- Validation of `global.get`/`global.set` instructions. [#351](https://github.com/wasmx/fizzy/pull/351)
+- Validation of `local.get`/`local.set`/`local.tee` instructions. [#363](https://github.com/wasmx/fizzy/pull/363)
+- Validation of stack height at the end of blocks and functions, at branch instructions, and after unreachable
+  instructions.  [#319](https://github.com/wasmx/fizzy/pull/319) [#356](https://github.com/wasmx/fizzy/pull/356)
+  [#343](https://github.com/wasmx/fizzy/pull/343) [#384](https://github.com/wasmx/fizzy/pull/384)
+  [#386](https://github.com/wasmx/fizzy/pull/386)
+- Validation of start section. [#353](https://github.com/wasmx/fizzy/pull/353)
+- The `.inputs` files to `fizzy-bench` now have to include signature of the called function.
```
Comment (gumb0) [inline CHANGELOG.md:41]: Maybe
```suggestion
- `fizzy-bench` now requires `.inputs` files to have to include signature of the called function.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,9 +7,70 @@ and this project adheres to [Semantic Versioning].
...
+
+- Exception error messages cleaned up. [#315](https://github.com/wasmx/fizzy/pull/315)
+- `fizzy-bench` now reports more useful error messages. [#316](https://github.com/wasmx/fizzy/pull/316)
+- Operand stack optimizations. [#247](https://github.com/wasmx/fizzy/pull/247)
+  [#265](https://github.com/wasmx/fizzy/pull/265) [#339](https://github.com/wasmx/fizzy/pull/339)
+- Optimization to resolve all jump destinations during parsing instead of having control stack in execution.
+  [#304](https://github.com/wasmx/fizzy/pull/304) [#312](https://github.com/wasmx/fizzy/pull/312)
+  [#341](https://github.com/wasmx/fizzy/pull/341) [#373](https://github.com/wasmx/fizzy/pull/373)
+  [#391](https://github.com/wasmx/fizzy/pull/391)
+- Better execution time measurement in fizzy-bench. [#383](https://github.com/wasmx/fizzy/pull/383)
```
Comment (gumb0) [inline CHANGELOG.md:64]:
```suggestion
- Better execution time measurement in `fizzy-bench`. [#383](https://github.com/wasmx/fizzy/pull/383)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,9 +7,71 @@ and this project adheres to [Semantic Versioning].
...
+- Validation of element section. [#350](https://github.com/wasmx/fizzy/pull/350)
+- Validation of export section. [#357](https://github.com/wasmx/fizzy/pull/357)
+- Validation of `global.get`/`global.set` instructions. [#351](https://github.com/wasmx/fizzy/pull/351)
+- Validation of `local.get`/`local.set`/`local.tee` instructions. [#363](https://github.com/wasmx/fizzy/pull/363)
+- Validation of stack height at the end of blocks and functions, at branch instructions, and after unreachable
+  instructions.  [#319](https://github.com/wasmx/fizzy/pull/319) [#356](https://github.com/wasmx/fizzy/pull/356)
+  [#343](https://github.com/wasmx/fizzy/pull/343) [#384](https://github.com/wasmx/fizzy/pull/384)
+  [#386](https://github.com/wasmx/fizzy/pull/386)
+- Validation of start section. [#353](https://github.com/wasmx/fizzy/pull/353)
+- `fizzy-bench` now requires `.inputs` files to have to include signature of the called function.
```
Comment (gumb0) [inline CHANGELOG.md:41]: typo
```suggestion
- `fizzy-bench` now requires `.inputs` files to include signature of the called function.
```

Comment (gumb0) [reply CHANGELOG.md]: still there?

**Comment by gumb0** [comment]
Looks good to me.

**Pull request merged**